opentelemetry-sdk/tests/resources/test_resources.py

```diff
@@ -93,13 +154,15 @@ def test_resource_merge_empty_string(self):
         the exception of the empty string.
 
         """
-        left = resources.Resource({"service": "ui", "host": ""})
+        left = resources.Resource({"service": "ui", "host": ""}, "")
         right = resources.Resource(
-            {"host": "service-host", "service": "not-ui"}
+            {"host": "service-host", "service": "not-ui"}, ""
         )
         self.assertEqual(
             left.merge(right),
-            resources.Resource({"service": "not-ui", "host": "service-host"}),
+            resources.Resource(
+                {"service": "not-ui", "host": "service-host"}, ""
+            ),
         )
 
     def test_immutability(self):
```

Comment:
Should you add a line to test if the schema_url should be immutable, since the [entire Resource should be immutable](https://github.com/tigrannajaryan/opentelemetry-specification/blob/fecb35adda061f9723d2f4cd527681eb84fb42c3/specification/resource/sdk.md)?